docs/modules/sort/quick_start.md

```diff
@@ -32,35 +30,17 @@ Notice：
 - `inlong-sort/sort-[version].jar` is the compiled jar
 
 ## Necessary configurations
-- `--cluster-id ` represent a specified inlong-sort application, same as the configuration of `sort.appName` in inlong-manager
-- `--dataflow.info.file` dataflow configuration file path
-- `--source.type` source of the application, currently "pulsar" is supported
-- `--sink.type` sink of the application, currently "clickhouse", "hive", "iceberg", "kafka" are supported
-- `--metrics.audit.proxy.hosts` audit proxy host address for reporting audit metrics
+- `--group.info.file` dataflow configuration file path
+- `--lightweight` whether to use lightweight sort,default false
```

Comment:
we will use the lightweight version all the time, `--lightweight` is useless?

Reply:
Yes,I will do it latter

Reply:
done